components/channel_groups_manage_modal/channel_groups_manage_modal.jsx

```diff
@@ -54,7 +60,30 @@ export default class ChannelGroupsManageModal extends React.PureComponent {
         this.props.actions.openModal({modalId: ModalIdentifiers.ADD_GROUPS_TO_TEAM, dialogType: AddGroupsToChannelModal});
     };
 
+    makeChannelAdmin = async (item, listModal) => {
+        this.props.actions.patchGroupSyncable(item.id, this.props.channel.id, Groups.SYNCABLE_TYPE_CHANNEL, {scheme_admin: true}).then(async () => {
+            listModal.setState({loading: true});
+            const {items, totalCount} = await listModal.props.loadItems(listModal.setState.page, listModal.state.searchTerm);
+            listModal.setState({loading: false, items, totalCount});
+        });
+    };
+
+    makeChannelMember = async (item, listModal) => {
+        this.props.actions.patchGroupSyncable(item.id, this.props.channel.id, Groups.SYNCABLE_TYPE_CHANNEL, {scheme_admin: false}).then(async () => {
+            listModal.setState({loading: true});
+            const {items, totalCount} = await listModal.props.loadItems(listModal.setState.page, listModal.state.searchTerm);
+            listModal.setState({loading: false, items, totalCount});
+        });
+    };
+
```

Comment:
`makeChannelAdmin` and `makeChannelMember` are almost identical except for a `true`/`false` boolean. Does it make sense to combine them into a single function?